ballerina/tests/constriant_error_message_test.bal

```diff
@@ -161,8 +161,8 @@ Trainee TRAINEE = {
 };
 
 @test:Config {}
-function testErrorMessageInRecordTypePositive() {
-    Trainee trainee = TRAINEE.clone();
+isolated function testErrorMessageInRecordTypePositive() returns error? {
+    Trainee trainee = check TRAINEE.cloneWithType();
```

Comment:
Do we need to clone this record, because it is readonly?

Reply:
For this test we do not need. I think I just replace all the places where I use `TRAINEE.clone()`. Will change this 👍🏼

Reply:
Fixed via [b64795f](https://github.com/ballerina-platform/module-ballerina-constraint/pull/105/commits/b64795fb2ee17462239f31a7ac6a0f9bd8ba889d)